pkgs/development/tools/crosstool-ng/default.nix

```diff
@@ -0,0 +1,55 @@
+{ pkgs, stdenv }:
```

Comment:
It is generally preferred to take each dependency as an argument, instead of the whole ```pkgs``` package set.